src/appshell/view/appmenumodel.cpp

```diff
@@ -21,6 +21,9 @@
  */
 #include "appmenumodel.h"
 
+#include <QtDBus/QDBusConnection>
+#include <QtDBus/QDBusConnectionInterface>
```

Comment:
I think these would also need to be wrapped in `#ifdef Q_OS_LINUX`

Reply:
Yeah, I've forgotten about those, I'll fix that...